webapp/channels/src/components/integrations/abstract_command.jsx

```diff
@@ -437,7 +437,7 @@ export default class AbstractCommand extends React.PureComponent {
                                         values={{
                                             link: (
                                                 <ExternalLink
-                                                    href='https://developers.mattermost.com/integrate/admin-guide/admin-slash-commands/#built-in-commands'
+                                                    href='https://developers.mattermost.com/integrate/slash-commands/built-in/'
```

Comment:
Is this meant to be a perma-link?

Reply:
aha! Ideally yes. @emdecr - Can we create permalinks that point to subtopics on a page? If so, can you please create one for this topic as ``/pl/built-in-slash-commands?

Reply:
Correct and thank you!!

Reply:
Corrected via this PR.